### PR TITLE
Update Testing section docs

### PR DESCRIPTION
**Pull request by aldeed**
To reflect current guidelines and tools

### CODE REVIEW

File: developer/testing/creating-test-data-meteor.md [hunk 1]
```diff
@@ -46,3 +46,35 @@ The following factories are predefined. The appropriate object is created by exe
...
   - `product` creates empty product without variants
+
+## Faker
+
+The [reaction-factories](https://github.com/reactioncommerce/reaction-factories) package supplies `Factory` and `faker` with additional methods for testing Reaction packages.
+
+**The recommended way to create shop:**
+
+```js
+const shopId = ReactionFaker.shops.getShop()._id;
```
Comment (zenweasel) [inline developer/testing/creating-test-data-meteor.md]: Pretty sure this ReactionFaker stuff is all gone and just comes from the Fixtures stuff

File: developer/testing/running-acceptance-tests.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
+## Running Acceptance Tests
```
Comment (zenweasel) [inline developer/testing/running-acceptance-tests.md]: This stuff can all be removed. It was removed from the project a while ago.

File: developer/testing/creating-test-data-meteor.md (hunk 1 above)
Comment (aldeed) [reply developer/testing/creating-test-data-meteor.md]: OK, this is one article I didn't read through. I'll check it out and update.

**Comment by aldeed** [comment]
@zenweasel I made those changes, and finished the rest of the stuff that was TODO.

**Pull request merged**